mappings/net/minecraft/entity/ai/brain/task/StayCloseToTargetTask.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/unmapped/C_xwbaenjp net/minecraft/entity/ai/brain/task/StayCloseToTargetTask
+	FIELD f_foxifxbl speedModifier F
+	FIELD f_tgpvzaov targetPositionGetter Ljava/util/function/Function;
+	FIELD f_tjnvchuh closeEnough I
```

Comment:
isCloseToTarget?

Reply:
It's not a predicate, it's the distance considered "close enough" to the target

Reply:
distanceToEntity id assume since it is not a predicate

Reply:
How about `distanceThreshold` or something similar?

Reply:
Well threshold implies something activates when it meets that requirement. If thats the case then sure

Reply:
I havent looked at the code so I cant tell if it activates something or its just the general distance to the entity

Reply:
`requiredDistanceToTarget`?

Reply:
```suggestion
	FIELD f_tjnvchuh requiredDistanceToTarget I
```